packages/sandbox/src/file_watching_sandbox.ts

```diff
@@ -133,6 +134,12 @@ export class FileWatchingSandbox extends EventEmitter implements Sandbox {
 
     await this.printSandboxNameInfo(options.identifier);
 
+    if (options.functionStreamingOptions?.logsOutFile) {
+      this.functionsLogStreamer.setOutputLocation(
+        options.functionStreamingOptions.logsOutFile
+      );
+    }
```

Comment:
This seems like something that should be set in the ctor of the LambdaFunctionLogStreamer

Reply:
None of the sandbox options are provided to the `SandboxSingletonFactory` like `outDir`. I have changed it to pass this option in the start/activate like the `BackendDeployer` instead of just a method for `setOutputLocation`